frontend/src/app/modules/players/players-selectors.js

```diff
@@ -62,3 +62,18 @@ const getPlayerCriteriaPoints = (player, matchesLast, criteria) => {
       return player.rating
   }
 }
+
+export const getRankingsForPlayer = createSelector(
+  getTopPlayers,
+  (state, playerId) => playerId,
+  (players, playerId) => getRatingStatisticsForPlayer(players, playerId),
```

Comment:
Unfortunatelly not, as there would be some problem with calling `.apply` during memoization that I don't really understand. I think it is because `getRatingStatisticsForPlayer` is a function and not a selector

Reply:
tricky one :) I realized that it happens because getRatingStatistics is undefined when you create a selector... try to define getRatingStatistics before creation of selector.

Reply:
Thanks, that did the thing.